src/components/BookContainer.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import BookContent from './Book';
+import { useSelector } from 'react-redux';
 import AddBook from './AddBook';
+import BookContent from './Book';
 
 const BookContainer = () => {
-  const books = [
-    {
-      title: 'how to become a milionaire',
-      author: 'shepherd bushiri',
-      id: 1,
-      category: 'Novel',
-    },
-    {
-      title: 'parach daily',
-      author: 'shepherd bushiri',
-      id: 2,
-      category: 'Novel',
-    },
-  ];
-
+  const data = useSelector((state) => state.booksReducer);
+  const books = data;
   return (
     <div className="main-container">
+      {/* {books.map((book) => (
+        <div key={book.id}>
+          <li className="books-content">
+            {book.title}
+            {' by '}
+            {book.author}
+            {' - '}
+            {book.category}
+          </li>
+          <button type="button" id={book.id} onClick={
+            () => dispatch(deleteBook(book.id))
+          }>Remove</button>
+        </div>
+      ))} */}
```

Comment:
- [ ] Kindly remove all unnecessary commented-out code in this file. Removing such codes makes your code clean and it shows that you adopt best JavaScript practices when building an application. 👍